### DISCUSSION TITLE
Multiple networks for a single proposal / space

### DISCUSSION THREAD

**Discussion by metrox-eth** [snapshot-labs/snapshot#718] [Ideas]
_Initially started by Fabien on Canny
March 20, 2021_

Give ability to do vote cross-network.

Comments from Canny:

1. cake
Such as BCS, TRON and many more

2. Hamzah Khan
Definitely an important feature for an increasing cross-chain world.Polygon teams would love to use this and engage communities.

3. Capn Cook
Its an important feature requested by lots of teams.

**Comment by bonustrack** [comment]
It's possible to use multiple network using the strategy "multichain" here is the strategy:
https://snapshot.org/#/strategy/multichain
You can test it here https://snapshot.org/#/playground/multichain
